recipe/meta.yaml

```diff
@@ -65,3 +73,4 @@ extra:
     - wolfv
     - tkralphs
     - jschueller
+    - brynpickering
```

Comment:
@brynpickering since I based my PR off of yours, I kept this until now.
Before we merge this, I want to confirm: Do you want to stay in the maintainers list?

Reply:
Thanks for checking @jpfeuffer. I'm happy to not be included. I did very little compared to what you've now done to make it work (yay!). I'll put myself in if I add anything substantive in future.